metadata-service/auth-impl/src/main/java/com/datahub/authorization/PolicyFetcher.java

```diff
@@ -39,10 +39,15 @@ public class PolicyFetcher {
 
   public PolicyFetchResult fetchPolicies(int start, int count, Authentication authentication)
       throws RemoteInvocationException, URISyntaxException {
+    return fetchPolicies(start, count, "*", authentication);
```

Comment:
Should we do "" here? Want to remove star since its an elasticsearch thing

Reply:
Okay sure.